README.md

```diff
@@ -76,7 +76,9 @@ Before using the base profiles functionality, ensure that you have [Zowe CLI](ht
    zowe auth login apiml
    ```
 
-2. Follow the instructions and provide all necessary information to create a base profile. 
+2. Follow the instructions to complete the authentication process and provide all necessary information to create a base profile with a token.
```

Comment:
Can it be what I had in my suggestion?

Reply:
2. Follow the steps to complete the login.
   A local base profile with token is created.